framework/elements/BaseElement.js

```diff
@@ -0,0 +1,75 @@
+class BaseElement {
+    constructor(name, selector) {
+        this.name = name;
+        this.selector = selector;
+    }
+
+    findElement() {
+        return browser.$(this.selector);
+    }
+
+    findElements() {
+        return browser.$$(this.selector);
+    }
+
+    moveTo() {
+        const element = this.findElement();
+        element.moveTo();
+    }
+
+    isDisplayed() {
```

Comment:
you may want to add possibility to wait until the result is true by using some of waitFor methods. this is applicable to some other methods below like isEnabled, etc